requirements.txt

```diff
@@ -1,3 +1,12 @@
+bertopic==0.16.2
+ipython==8.12.3
+keybert==0.8.4
+langchain==0.1.20
+langchain_community==0.0.38
+langchain_openai==0.1.7
+numpy==1.26.4
+openai==1.30.1
 pandas==2.2.2
 python-dotenv==1.0.1
-IPython==8.12.3
+scikit_learn==1.4.2
+tiktoken==0.6.0
```

Comment:
There is some issue with `tiktoken` package version. 

```
ERROR: Cannot install -r requirements.txt (line 6) and tiktoken==0.6.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested tiktoken==0.6.0
    langchain-openai 0.1.7 depends on tiktoken<1 and >=0.7

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

Reply:
is there a reason you chose that particular version. Can we just use that is being used by `langchain-openai`?

Reply:
Remove it the exclusive pkg declaration and just import since it is already available package in the venv

Reply:
I think I will resolve it